index.js

```diff
@@ -75,11 +75,13 @@ LiveReloadPlugin.prototype.failed = function failed() {
 };
 
 LiveReloadPlugin.prototype.autoloadJs = function autoloadJs() {
+  // Random alphanumeric string appended to id to allow multiple instances of live reload
+  var instanceId = Math.floor(Math.random() * Math.pow(32,6)).toString(32);
```

Comment:
Can you store this as `this.instanceId` in the constructor? It'll ensure the id stays the same across multiple entry points using the same plugin instance.